environment.yml

```diff
@@ -37,6 +37,7 @@ dependencies:
   - natsort
   - nc-time-axis
   - netCDF4
+  - numba
```

Comment:
it would be great if you didn't use `numba` please - we've had numerous problems with `numba` wrt dependency agreement and environment solving, they don't really offer great support. Sorry to have started with a negative remark :grin:

Reply:
> Hi @mo-abodas welcome to the Tool, and cheers for your contributions! I can definitely have a look at the code from a technical standpoint, so you can count on me on the technical review side - more frequent scientific reviewers are good folk like @alistairsellar or @axel-lauer or @LisaBock grin

Thanks @valeriupredoi , @ehogan has also volunteered to do the technical review, but extra eyes are always welcome!

Reply:
No worries, I didn't know that, I won't use it. A real shame though, as it gives a massive speed up!

Reply:
Cheers! I'll have a look at the used case and try see how to speed up the code without numba, there's a few other ways, plus, don't just want to be a naysayer without suggesting alternatives 😁

Reply:
actually @mo-abodas I'll take back my reservations wrt numba in the env - we actually bring it in from somewhere (some dependency needs it) so why not add it back as direct dependency, it seems it's been behaving OK lately, see the env in the [github action test](https://github.com/ESMValGroup/ESMValTool/actions/runs/5144236584/jobs/9260276712)